pkg/pillar/types/physicalioadapters.go

```diff
@@ -24,8 +23,19 @@ type PhysicalAddress struct {
 }
 
 // PhyIOUsagePolicy - Usage policy for the Adapter
+// This is constructed from
 type PhyIOUsagePolicy struct {
 	FreeUplink bool
+	// FallBackPriority
+	//  0 is the highest priority.
+	//  Lower priority interfaces are used only when NONE of the higher
+	//  priority interfaces are up.
+	//  For example:
+	//      First use all interfaces with priority 0
+	//      if no priority 0 interfaces, use interfaces with priority 1
+	//      if no priority 1 interfaces, use interfaces with priority 2
+	//      and so on..
+	FallBackPriority uint32
```

Comment:
What does this change have to do with making protobuf compile?

Reply:
Keeping the fields consistent.. FallBackPriority is defined in zconfig.PhyIoUsagePolicy

Reply:
OK